library/src/main/java/org/xmtp/android/library/extensions/Date.kt

```diff
@@ -0,0 +1,6 @@
+package org.xmtp.android.library.extensions
+
+import java.util.*
+
+val Date.millisecondsSinceEpoch: Double
+    get() = (System.currentTimeMillis() * 1000).toDouble()
```

Comment:
Any reason this is double vs `createdNs` using Longs below?

Reply:
I ended up just removing this entirely. This was something I brought over from iOS and now that I look at it it we don't need it in android. currentTimeMillis is already in millis so we don't have to convert it to millis.